### PR TITLE
feat(components): Export FormatValue component

### PR DESCRIPTION
**Pull request by gjou-tlnd**
**What is the problem this PR is trying to solve?**

Escaping blank characters is needed in multiple places, we should reuse the `FormatValue` component  instead of copying it's implementation
Next step : remove it's duplicates

https://github.com/Talend/ui/blob/master/packages/datagrid/src/components/DefaultCellRenderer/FormatValue.component.js

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by romainseb — APPROVED** [review]

File: packages/components/src/FormatValue/FormatValue.component.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
```
Comment (trob-tlnd) [inline packages/components/src/FormatValue/FormatValue.component.js:2]: this file will be in jsx, no ?

Comment (gjou-tlnd) [reply packages/components/src/FormatValue/FormatValue.component.js:2]: no

Comment (gjou-tlnd) [reply packages/components/src/FormatValue/FormatValue.component.js:2]: Talend UI doesn't use jsx extension

Comment (trob-tlnd) [reply packages/components/src/FormatValue/FormatValue.component.js:2]: ok

**Review by trob-tlnd — APPROVED** [review]

**Pull request merged**